memfs/memory.go

```diff
@@ -69,21 +75,40 @@ func (fs *Memory) OpenFile(filename string, flag int, perm os.FileMode) (billy.F
 	return f.Duplicate(filename, perm, flag), nil
 }
 
+func (fs *Memory) resolvePath(path string) string {
+	fullpath := clean(fs.Join(fs.base, path))
+	l, ok := fs.links[fullpath]
+	if !ok {
+		return fullpath
+	}
+
+	// IsAbs requires an unit so we check to if the path stars with a os.PathSeparator
```

Comment:
hehe, even for me was obscure the meaning. Unit refers to a windows unit, so means that in Windows. To be Abs a path should contains a `C:\`, the unit. 

And yes is a logical short-circuit.

New comment 
```
// On Windows OS, IsAbs validates if a path is valid based on if stars with a
// unit (eg.: `C:\`)  to assert that is absolute, but in this mem implementation
// any path starting by `separator` is also considered absolute
```